Microsoft.Identity.Web/AuthorityHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Identity.Web
+{
+    public static class AuthorityHelpers
+    {
+        internal static bool IsV2Authority(string authority)
+        {
+            if (string.IsNullOrEmpty(authority))
+                return false;
+
+            return authority.EndsWith("/v2.0");
+        }
+
+        internal static string BuildAuthority(MicrosoftIdentityOptions options)
```

Comment:
Added a helper to build authority, case it is not present in appsettings. It takes into consideration AAD and B2C authority patterns.

Reply:
are there tests for this? 

---
In reply to: [371537589](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/272#discussion_r371537589) [](ancestors = 371537589)